lib/service/discovery.go

```diff
@@ -149,6 +149,10 @@ func (process *TeleportProcess) integrationOnlyCredentials() bool {
 	return process.Config.Auth.Enabled && modules.GetModules().Features().Cloud
 }
 
+// accessGraphClusterSNI is a constant representing the SNI (Server Name Indication) prefix value
+// used when connecting to Access Graph.
+const accessGraphClusterSNI = "access-graph.cluster."
```

Comment:
Please don't use sni otherwise all certificates must be updated in cloud and customers. Use the next protos to send it